docs/multi-homing.md

```diff
@@ -88,6 +97,24 @@ spec:
     }
 ```
 
+#### Network Configuration reference
+- `name` (string, required): the name of the network. This attribute is **not** namespaced.
+- `type` (string, required): "ovn-k8s-cni-overlay".
+- `topology` (string, required): "layer2".
+  `subnets` (string, optional): a comma separated list of subnets. When multiple subnets
+  are provided, the user will get an IP from each subnet.
+- `mtu` (integer, optional): explicitly set MTU to the specified value. Defaults to the value chosen by the kernel.
+- `netAttachDefName` (string, required): must match <namespace>/<net-attach-def name>
+  of the surrounding object.
+- `excludeSubnets` (string, optional): a comma separated list of CIDRs / IPs.
```

Comment:
Could you have a sample of excludeSubnets configuration in NetworkAttachmentDefinition?

Reply:
Done.